systems/subscriber/sim-manager/pkg/server/sim_manager.go

```diff
@@ -118,23 +118,18 @@ func (s *SimManagerServer) AllocateSim(ctx context.Context, req *pb.AllocateSimR
 			"invalid networkId: subscriber is not registered on the provided network")
 	}
 
-	packageID, err := uuid.FromString(req.GetPackageId())
+	packageId, err := uuid.FromString(req.GetPackageId())
 	if err != nil {
 		return nil, status.Errorf(codes.InvalidArgument,
 			"invalid format of package uuid. Error %s", err.Error())
 	}
 
-	packageInfo, err := s.packageClient.Get(packageID.String())
+	packageInfo, err := s.packageClient.Get(packageId.String())
 	// think about how to handle different types of rest errors
 	if err != nil {
 		return nil, err
 	}
 
-	if packageInfo.OrgId != remoteSubResp.Subscriber.OrgId {
```

Comment:
We need to check `packageinfo.OrgId` against `s.OrgId`, so this line should not be removed but should be updated